openedx/core/djangoapps/content_libraries/tests/test_content_libraries.py

```diff
@@ -1101,10 +1101,6 @@ def test_invalid_key(self, endpoint, endpoint_parameters):
             endpoint.format(**endpoint_parameters),
         )
         self.assertEqual(response.status_code, 404)
-        msg = f"XBlock {endpoint_parameters.get('block_key')} does not exist, or you don't have permission to view it."
-        self.assertEqual(response.json(), {
-            'detail': msg,
-        })
```

Comment:
Because I moved the usage key parsing into Django's URL router, we still get a 404 response here but it's not a customized one for this API anymore.